GVFS/GVFS.UnitTests/Maintenance/GitMaintenanceStepTests.cs

```diff
@@ -0,0 +1,110 @@
+﻿using GVFS.Common;
+using GVFS.Common.FileSystem;
+using GVFS.Common.Git;
+using GVFS.Common.Maintenance;
+using GVFS.Common.Tracing;
+using GVFS.Tests.Should;
+using GVFS.UnitTests.Mock.Common;
+using GVFS.UnitTests.Mock.FileSystem;
+using NUnit.Framework;
+
+namespace GVFS.UnitTests.Maintenance
+{
+    [TestFixture]
+    public class GitMaintenanceStepTests
+    {
+        [TestCase]
+        public void GitMaintenanceStepRunsGitAction()
+        {
+            ITracer tracer = new MockTracer();
+            GVFSEnlistment enlistment = new MockGVFSEnlistment();
+            PhysicalFileSystem fileSystem = new MockFileSystem(new MockDirectory(enlistment.EnlistmentRoot, null, null));
+
+            GVFSContext context = new GVFSContext(tracer, fileSystem, null, enlistment);
+            GitObjects gitObjects = new MockPhysicalGitObjects(tracer, fileSystem, enlistment, null);

```

Comment:
Move these 5 lines of code to a `GVFSContext InitContext(out GitObjects gitObjects)` method to use in all the tests?